src/AddTodoForm.js

```diff
@@ -1,12 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = useState(''); 
+
+    function handleAddTodo(event) {
+        event.preventDefault();
+        const newTodo = {
+            title: todoTitle,
+            id: Date.now()
+        };
+        onAddTodo(newTodo);
+        console.log(todoTitle);
+        setTodoTitle('');
+    }
+
+
+    function handleTitleChange(event) {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
```

Comment:
Same thing here, it's perfectly fine as-written but could be simplified to this
```suggestion
        setTodoTitle(event.target.value);
```